package.json

```diff
@@ -8,6 +8,8 @@
   "dependencies": {
     "bootstrap": "^3.3.4",
     "crypto-js": "^3.1.2-5",
+    "es5-shim": "^4.1.7",
```

Comment:
The ES6 shim adds objects like `Set`. The shim for ES5 polyfills methods like `Array.map` and `Object.keys`.

- ES5 shim: https://github.com/es-shims/es5-shim
- ES6 shim: https://github.com/paulmillr/es6-shim/